### DISCUSSION TITLE
Content organisation for multiple languages

### DISCUSSION THREAD

**Discussion by rkranendonk** [GitbookIO/community#621] [1. How to] [answered]
Hi all,

My goal is to create a product manual in different languages.

What is the best practice for this?

I was thinking of creating a separate space in GitBook for each language version, and then have each space sync to it's own folder in my Github repo (under ./docs).
Is this a good approach?

**Comment by addisonschultz** [comment, marked as answer]
Yes, it's definitely a good approach! I would actually recommend to check out this page in our docs - we outline this exact use case!

https://docs.gitbook.com/published-documentation/share/collection-publishing

**Reply by rkranendonk** [reply]
Thanks!
